src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -101,6 +101,19 @@ namespace EnergyPlus {
 
 static gio::Fmt fmtLD( "*" );
 
+// Integer constants for different system types handled by the routines in this file
+int const ParallelPIUReheatNum( 1 );
+int const SeriesPIUReheatNum( 2 );
+int const HeatingCoilWaterNum( 3 );
+int const BBWaterConvOnlyNum( 4 );
+int const BBSteamRadConvNum( 5 );
+int const BBWaterRadConvNum( 6 );
+int const FourPipeFanCoilNum( 7 );
+int const OutdoorAirUnitNum( 8 );
+int const UnitHeaterNum( 9 );
+int const UnitVentilatorNum( 10 );
+int const VentilatedSlabNum( 11 );
+	
```

Comment:
Thanks @RKStrand, I wasn't clear enough.  I was hoping for an enum class instead of stray constants.  Like:

```
enum class LocalEquipTypes {
  ParallelPIUReheat = 1;
  ...
};
```

Then using it like:

```
  case LocalEquipTypes::ParallelPIUReheat
```

I'll probably make this change before I do final testing/merge.  It will be nicer than just the list of constants.